### PR TITLE
[doc] Update own-instance.rst

### PR DESCRIPTION
**Pull request by dwitterer**
Improved English grammar and punctuation for professionalism.

## What does this PR do?

Improves the English grammar and sentence structure of this documentation page.

## Why is this change important?

This PR enhances the professionalism and readability of the modified page and makes this page more clear for new users.

## How to test this PR locally?

N/A, documentation change.

## Author's checklist

N/A

## Related issues

N/A, will check for duplicates now.
UPDATE: No duplicates, I'm the first to update this page :)

### CODE REVIEW

**Review by unixfox — APPROVED** [review]

File: docs/own-instance.rst [hunk 1]
```diff
@@ -80,7 +80,7 @@ Always use an instance which is operated by people you trust.  The privacy
 features of SearXNG are available to users no matter what kind of instance they
 use.
 
-If someone is on the go or just wants to try SearXNG for the first time public
-instances are the best choices.  Additionally, public instance are making a
-world a better place, because those who cannot or do not want to run an
-instance, have access to a privacy respecting search service.
+For those on the go, or just wanting to try SearXNG for the first time, public
+instances are the best choice.  Additionally, public instance are making the
+world a better place by giving those who cannot, or do not, want to run an
```
Comment (return42) [inline docs/own-instance.rst]: > "those who cannot, or do not, want to run"

I can't judge, but is this punctuation really correct in English? Why not..

> "those who can't or don't want to run"

File: docs/own-instance.rst [hunk 2]
```diff
@@ -2,62 +2,62 @@
...
 instance.  This means that the user of the public instance does not know whether
-their requests are logged, aggregated and sent or sold to a third party.
+their requests are logged, aggregated, and sent or sold to a third party.
 
-Also, public instances without proper protection are more vulnerable to abusing
-the search service, In this case the external service in exchange returns
-CAPTCHAs or bans the IP of the instance.  Thus, search requests return less
+Also, public instances without proper protection are more vulnerable to abuse of
+the search service which may cause the external service to enforce
+CAPTCHAs, or to ban the IP address of the instance.  Thus, search requests would return less
```
Comment (return42) [inline docs/own-instance.rst]: > "to enforce CAPTCHAs, or to ban the IP address"

is the additional comma before the “or” necessary?

**Review by return42 — COMMENTED** [review]
Thank you for improving SearXNG's documentation / my English is not good enough to correct a professional text, but I noticed two things that I would have expected differently .. see my comments below:

**Review by Bnyro — APPROVED** [review]

**Comment by return42** [comment]
@dwitterer I like to merge this PR / but first please give short feedback to my questions from my review / thanks :+1:

File: docs/own-instance.rst (hunk 1 above)
Comment (dwitterer) [reply docs/own-instance.rst]: Hi, sorry, was a bit busy. I'm not a linguist but I'm pretty sure both are correct/acceptable :) I'd keep it with the commas though, as I think it's a subjunctive clause and it flows better. I **_did_** make an oopsie though as it should be:

"Those who cannot, or do not want to, run" - moving the comma :)

File: docs/own-instance.rst (hunk 2 above)
Comment (dwitterer) [reply docs/own-instance.rst]: I think you're right in this case actually. The comma before "or" can be omitted, but we should add one between "service" and "which" as it's a "non-restrictive" clause. I'll see if I can edit the PR.

File: docs/own-instance.rst (hunk 1 above)
Comment (dwitterer) [reply docs/own-instance.rst:85]: I'd keep it with the commas though, as I think it's a subjunctive clause and it flows better.

**Comment by dwitterer** [comment]
@return42 See my responses above. I've also pushed some changes to my fork to fix a couple things, including what you said, so hopefully the PR should have updated to reflect this. If you click on 'commits' then it shows up for me :) Let me know if you are happy with this new version.

**Pull request merged**